exercises/saddle-points/example.rs

```diff
@@ -0,0 +1,23 @@
+
+pub fn find_saddle_points(input: Vec<Vec<u64>>) -> Vec<(u64, u64)>{
+    let mut saddle_points: Vec<(u64, u64)> = Vec::new();
+
+    let row_index = input.len();
+    let column_index = input[0].len();
```

Comment:
> Although this is indeed an index, it is also not the index of any existing element in the input (it is the index of the first element beyond it), so to avoid confusion I would call this `height` instead.

Same, except now `width`.